src/RefractDataStructure.cc

```diff
@@ -819,8 +854,7 @@ namespace drafter {
         }
 
         std::string description;
-        std::string& descriptionRef = description;
-        Join join(descriptionRef);
+        Join join(std::ref(description));
```

Comment:
There were something in earlier version just place string into was not working.
We can actually remove